### DISCUSSION TITLE
v0.16 generating core failed: unable to load gqlgen-test/graph/model - make sure you're using an import path to a package that exists

### DISCUSSION THREAD

**Discussion by abankspdx** [99designs/gqlgen#2000] [Q&A] [answered]
### What happened?
I followed the "getting started" directions in the README/on the website. I got:

```
➜  gqlgen-test go run github.com/99designs/gqlgen init
Creating gqlgen.yml
Creating graph/schema.graphqls
Creating server.go
Generating...
reloading module info
generating core failed: unable to load gqlgen-test/graph/model - make sure you're using an import path to a package that exists

Exec "go run ./server.go" to start GraphQL server
```

### What did you expect?
`server.go` to be generated

### Minimal graphql.schema and models to reproduce
I didn't provide one.

### versions
 - `gqlgen version`?  `v0.16`
 - `go version`? ``go version go1.16.6 darwin/amd64`
 - dep or go modules? modules

Also I tried my best to follow the info from: https://github.com/99designs/gqlgen/issues/1860, but it seems like `init` is the only way to generate `server.go`, and so updating the `gqlgen.yml` to remove the autobind lines didn't fix the problem. It seems like `init` generates `server.go`, and I can't rerun `init` if I already have a `gqlgen.yml`? Not really sure how to proceed. Never used this tool before.

**Comment by frederikhors** [comment]
Please follow the instructions. This is already fixed.

**Reply by abankspdx** [reply]
```
➜  svc mkdir example
➜  svc cd example
➜  example go mod init example
go: creating new go.mod: module example
➜  example printf '// +build tools\npackage tools\nimport _ "github.com/99designs/gqlgen"' | gofmt > tools.go
➜  example go mod tidy
go: finding module for package github.com/99designs/gqlgen
go: found github.com/99designs/gqlgen in github.com/99designs/gqlgen v0.16.0
➜  example go run github.com/99designs/gqlgen init
Creating gqlgen.yml
Creating graph/schema.graphqls
Creating server.go
Generating...
reloading module info
generating core failed: unable to load example/graph/model - make sure you're using an import path to a package that exists

Exec "go run ./server.go" to start GraphQL server
➜  example
```

This is exactly from the readme: https://github.com/99designs/gqlgen#quick-start

**Reply by jangelon** [reply]
@frederikhors 

I think the issue is that people are following the init command that's listed in the [quickstart instructions](https://github.com/99designs/gqlgen#quick-start).
When they use the init command, the config is autogenerated by the init, as you know, so the user doesn't have the chance to comment out the autobind.

In the v.0.16.0 release `init` command you can see the autobind is not commented out when it is created.
https://github.com/99designs/gqlgen/blob/v0.16.0/cmd/init.go#L57-L58

This is currently fixed in the master branch, and commented out there, but most users are getting the release, and run into the issue.

Once the init runs and fails, the user can comment out the autobind, but by then the server.go has not been created.
Running init again at this point will fail saying that `gqlgen.yml already exists`
Running generate at this point will not create the server.go file.
The user is now stuck without a server.go file while trying to execute the [quickstart](https://github.com/99designs/gqlgen#quick-start) listed in the docs.

At this point most new users will probably move on to another package, which is unfortunate.

I think a minor release that just comments out that field in the init built config would fix the frustrations.

**Reply by abankspdx** [reply]
Yes this is exactly what I'm encountering and how I feel. Thanks for summarizing @jangelon !

**Reply by frederikhors** [reply]
@mtibben, @StevenACoffman are we ready for 0.16.1 that fixes this too?

**Reply by frederikhors** [reply]
Thank you @jangelon.

**Reply by ROCKWAR** [reply]
I solved this commenting on the "autobind" in the GQLGEN.YML file

**Comment by abankspdx** [comment, marked as answer]
This seems to be resolved with using the new gqlgen version. Thanks!
